libafl/src/mutators/token_mutations.rs

```diff
@@ -1583,10 +1583,10 @@ where
 
                             // Compare v1 against v0
                             rtn_found |= self.rtn_extend_encoding(
-                                orig_v1,
-                                orig_v0,
-                                new_v1,
-                                new_v0,
+                                orig_v1.as_slice(),
```

Comment:
@addisoncrump can we do some `AsRef<>` trickery here to make this cleaner?

Reply:
Hm... potentially, but I'm not entirely sure. AsSlice is its own thing, you might have something that impls AsSlice differently than AsRef<[T]>.